app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_secure_password :password
+  has_secure_password :recovery_password, validations: false
+  validates :email, presence: true, uniqueness: true
+  validates :notifications, inclusion: { in: [true, false ]}
+
+  def condensed 
```

Comment:
I don't think this name is very descriptive. Condensed what? I think a better names would include `condensed_json`, `truncated_json`, `safe_json`, etc. I'll let you decide on the best choice of name though.